### PR TITLE
Variable header

### PR DESCRIPTION
**Pull request by kwinto**
This PR implements a variable height header. 

Ref: https://cognigy.visualstudio.com/Cognigy.AI/_workitems/edit/55733

Beside that:
- Fixes the background color as per Figma spec.
- Extends Webchat API with "startConversation" method for customers to be able to bypass a homescreen when necessary
- Immidiately scrolls chat history down when opened.

To test it, you can add this url to the settings object of the initial config:
``` javascript
settings: {
    headerLogoUrl: `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAMgAAADECAYAAADamm7lAAAAAXNSR0IArs4c6QAAADhlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAAqACAAQAAAABAAAAyKADAAQAAAABAAAAxAAAAAB9Y/KuAAAdd0lEQVR4Ae1dCZAUVba93V1VvW+0LAIDggiIKAgCooAoIILruCGiMOM2MaHxBf3h/44BLoT6v/EdMEZjYhx1xgURR/9ndAQXcEEQFUFARUAEQUAWm9736ur/TrdVZFVnVeVyMysz692IiqrKynzvvvPuqXz53n33ZrQJISnsCNQ2hOjA0Vbaf6TjdaSilSpqQu2vyvb3NqprDFFzC1FLsI2axasl2KGG30cU8GWQX7wCfqL8nEwqLcygkkK8d7y6lWZR724dr15ds6ggN5O9DbJAogxJEHNmUFMfop37grRjX4t4BWn73hbafSBIlbX2/u+UFGRQ/14+GtzXT4P6+MTLTwPFe2GeJI6ZHpYE0Yne3kNB2ri9mTbuaKZNO1po1/5f/vZ1lmPX6QN6+2jEID+NHBSgkYMD1LeHuD1J0YyAJEgSqKpqQ7R2axOt2dxEH4vX0cpQkiuc/XPXkkwaPzybJojXuDOyqbhA3mES9ZgkiAo6Px4J0opPGmn1F420ZVcLhdzNCZUWdhzKFNwYNsBPk87Koenn5NCvusm7SyxYkiC/IHLw51ZBigZasb6RvvpePDmnoZx+sp+mjwVZcqnnCVlpiEDnJqc1QTBztHpDIy1bXU/rtjZ3RieNj5x7RoBmTMqjSaNy2mfU0hWKtCQIHrRffreelq9poGPVHh0/MVl0l6JMumJCLl1/YV5aPuCnFUE2iZmnZ9+so/fEXUOu/uhjUEYG0RRxN7n50nwxKxbQd7GLz/Y8QbAOumpDEz39Ri1t3pmezxbc9jl8oJ9uu6yAJo/Kpgwwx8PiaYJ8sKmRFi+roW17nL1W4Vb7GtLPR3NnFNL5I3Lc2oSkenuSIOvEusWiV2rap2iTIiBPMI0ApornXVdI54p1Fa+Jpwiy+2CQHn2hmj7c1OS1fnJFeyaOyKZ7ZxdR/57eWU/xBEGq60L05Gu19OLbdRRsdYUteVZJn1g+ufGifLrj6gIqynf/Kr3rCbJ8Tb24a9TI6VqHUQ7Tw/fOLhRTxHkO00yfOq4lyIGjQZr/1+p2/yh9TZZn24kA/L4W3lpEvbq6c9jlOoJg2vb5lfX0x6U11NBkr0u5nYblpbpyszPorpmFNGdanuumhV1FkEPHWumeJytp/dfSLcSNBBo7NECP3VFCPbq4x8/LNQR557MGuu8vVVRl80YkNxqik3UuFhu7Hv5dMU0dk+tkNSO6OZ4gTc1t9NDfqujV1Q0RpeUH9yNw7aRcWvDbYsoOOHsl3tEEwYP47Y9X0je7pYuI+ynRuQWn9ffTU3eXOPoB3rEEwS6+eYsrbN/b3bkb5RErEcBe+kVzS9t3N1pZj9GyHUmQZ4Rj4WNLaqTHrdFeddl18He8Z1Yh3SIcIJ0mjiJIa6iNHny2mpa+V+80nKQ+NiAwc0oe3X9zEWVlOue5xDEEQYyoOxdV0kdfSj8qG2zRsVWcd2Y2PTGvpD0WmBOUdARByqta6aaHj9G2H6RbuhOMItU6DDnJR8/d14XKilO/XpJygvxU3kpzFpbTnoPSyzDVhumk+vv1zKLn55fRiWWpJUlKCbLvcJBmP3SsPUSnkzpH6uIMBBBS9YUFXahP99T5caWMINi7ceOD5XSkQgZNcIY5OlOLbqWZ9OL9ZSnbY5ISgiAw28wF5XT4mCSHM83SWVp175JJSx8qS0lgO9t3tMDhEMMqSQ5nGaGTtYGtwGZgO3aLrQTBbNXsh8rbUwLY3VBZn7sRQBoJ2A5syE6xjSD1Yp3jpkeOydkqO3vXY3VhphM2BFuyS2whCFbI71xcKcPv2NWrHq4HIZxgS7ApO8QWgix8TkYasaMz06UORK2BTdkhlhPkuX/V0hIRB1eKRIATAdgUbMtqsZQg679uov9+qcbqNsjy0xQB2BZszEqxjCA/iXwbc8VY0avJZ6zsFFm2NgRgW7Ax2JpVYglBmlraxE7AChmryqpek+VGEED6CtgabM4KsYQgjzxfnbZZmqzoJFlmYgSQEQw2Z4WwEwR5/ZCcRopEwE4EYHOwPW5hJcjRyla698+V3DrK8iQCmhCA7cEGOYWNIIh4eM9TlVRRY81YkLPRsixvIgDbgw3CFrmEjSCYl167RUY85OoYWY4xBGCDnOtuLATBrsD/EVFIpEgEnIAAbBE2ySEsBHngmSqqa+S7rXE0TJaRvgjAFmGTHGKaICs+aaD3N1q7msnRUFlGeiEAm4RtmhVTBKltCNHCv1sz/2y2YfJ6iQBsEzZqRkwR5M//W0s/V5pTwIzy8lqJQCIEYJuwUTNimCCISPL3t+rM1C2vlQhYjgBsFLZqVAwTBNlkm43Xa1RfeZ1EQBcCsNH/etH4Y4ChgEOfb2uiVRu8+2A+crCf/nRXqa6OSHTyn/5R6+p4w6WFGfTW410TNVH3b3MWHqPvfrTnH/a9z5sINjt6iP487oYIsniZuXGdbjRtvgBpjLuW8EX0my1y87k5IHemCCbNiQe6088HrybrgM2+/KB+gugeYq0TeTs2fOvdFXOkL54yOkcT6FpPGtDbT+ecHtB6ujzPAgRgs7BdvaKbIIuXeXvFHCH4Az7+8Ps3TM3X2zfyfGYEjNiuLoJ89GUjbf7Ou+nQMgUaMyZZk/j+/JHZhAiBUlKHAGwXNqxHdPXY0//09rTulFE5dOIJ1gyOfVkZdP2F1pBPT4en+7l6bVgzQZBI8/Nt3n32gOHcMNVaA77mgjzyWcO/dLd7ze2HDetJCquZIM+86e2ZK+SjOHuo/lkOzT0jTsRM0LSxvBMAeuqX53YgoMeWNRHkoIgasXK9vrGb2zpjtpjatUPkw7odKCeuA7YMm9Yimgjyyqp6EepRS3HuPCc3O4Mun5Bri/IjBgVoUB9Dy0+26JcOlcCWYdNaJClBEAP19Q+0FaalQieec+V5uVSYlxQKNtVnT7PnbsWmsAcLgk1rie+b1CoQB9XrWaBm2jy7dMm5OVSQx7/W4kE7tqxJsGnYdjJJSpBXV3v77jF6CIY8/mQ4sf6el5NJV0+0Z0jHqrjHCtNi2wkJ8rMIofKhx/OWWz21G8+mZl4oh1nxsLHrOGwbNp5IEhLk7U8bPR1b94SSTJosFgdTIf17+mjcGdI/KxXYh+tEbF/YeCJJSJC3GPb0Jqo81b/B78pvgd+V1nbdYNPUslZ90vG8FUmWL+IS5LBImLhxh3f9rrJEy6+1yO9Kq6FNHJFNPcridoHWYuR5JhD4YnuzSCgbf5gVt3dWilsPY4A6E02w5tILx+RQjy6p9fvIEvssZkkvX2s6WGOpsHHYejyJSxArAgHHUyIVx1P1cB7b1mvOzxXDvNij8rudCLyfIOi1KkHqRBbRjeLW41UZ0NtnaPulFXiUFWfR9HPklK8V2GotE8Ms2LyaqBLk06+bqcWe7cJqOll+7IaLrPXa1duAWTYvVOrVz+vnw9Zh82qiSpA1m5OvMKoV5oZj+TnC72q8s/6xzxwYoFNPkuOsVNpPPJtPO4L8WqxgF+SqNjuV/UMIFCEldQh8HOem0MlSMOW1/0j8aa/UNYGnZs5dfV/uVL8tG9H00nG5VJQv/bOMYMdxzY/C5tWmezsRZNMOvk7nUJyzjLOHBugUEWGEQ4KtbfRviyo07ytIVmdOIIOuPt9Zz0bJdPba72q234kgGz1MEM6p3Xc+a6RD5SGRj5Fvnz5W9qWkDgE12+9MkO3eXD3vVppJk87i87t66e0OYvzj/Qa2FMQnneijCcOt3fabOvNzfs0bVWw/iiDINf3tD94kCJ49EFmEQ3bua6EvfgETebpXrjefhyKsl9OmoMN6pcM7bD8233oUQXbtD3pyay0iiSCiCJe89E70HpklMd/N1HOeuIP0tCj0kBm90uFabMUFB5QSRRD8M3pRpgq/q26lPH5XNfUh+ufH0XcMBCTTE0omEcaIgzvL4vBDiepP999iORBFkO17o9njFbA43cr/76MGqlfJx/jSO3wP65jNCsh1w5SYXywHogiyw4N3kIEigshZg/k2Ji2JQ4Q31zVQZa26P4/enkYA7YvPddZqv942uPX8WA5EEeT7A967g3CuUK//uol2H1RfRG0Sy0evvR/9bGLGSDinpM3okW7XxnIgQpCWYJtYSeT5B3QKqAW5GXTpOM6p3cQEePnderGHhicd9hkDAnRaf55FTaf0hxv0AAfAhbBECIJIc0x9Gy475e9Xib0W+SKCCIccEi44yfbIwF3hI8YgF7Md5nXMgaPTywAHlFEXI9bjRf+rmVP4HACXiruDluiSsVPAZgwC+0SKC3jWbszokW7XHjh6fBgdIYjyoBcAQUank3vxTAXhlrtMY3wwuE2byaqqxB7+WZzrN8qy5ef4CChvFhGCHK44zpr4l7rnF84g0fC7Kq/S9nyGWzSeRbhE+mdxIam9HCUXIgSprNFmANqrSd2ZPUQmpwvO4vNp0rvG8ZqI+9rUfPxBzwwSfbr7CNFPpNiHgJILEYJUeIgg14uohYgYwiHb97aI/fn6PAyqatsI6yJcIqd8uZDUVo6SC54jCCKEXHMB3yKb0Yduo9epdeH4Ydn0q248rjJq5ctj0QjEIQjPkCC6Kvu/TTs7h05gynEOv6s3YvyutLYIvlmbv+PZfCb9s7SiznNeRc1xLkTuIPVxwp7wVGlfKZx+V69/2EANTcfB0tsKzrvIVfDPkuuGervA0PlKLkQI0qxvmG2oYqsvGtzXR4gQwiVLTe4WXCliG5dX88wOlhRk0mVi37oU6xFQciFCEOXyuvUqWFMDp9/Vuq3x/a60at8sXNuw45BLOANOcOnkxXKUXPAMQQpFxiZEBuESvVO78erFCnxIpLHjkNNPDtCwAXKcxYFlojJUCaK8rSS62Km/YQ8FknFyyE/lrfT+Rp7gefDr+UBDqi+tesstuVqRMn6ekguRO4jx4pxxJeeKMyKVILkKl4QDPHCUN21sLpUW8vwRcOjj9TIiBHHzDMm4YQHqJzI2cUiz8LvifG6ATmu3NtOegzx7bbL9GSnPa8KBs5PLUHIhQpBUZloyCxan3xVScmn1u9Kj9xKTM2LKuq6bnEcZ8iaihIT1s5ILEYIExD+TG+XEEzJZfZU4h0NKPLGmopxfV/6m93Pvbj46f6T0z9KLm9bzlVyIEMStSVxmMfpdbRNxkb7cac2CUG19G72xNn4mI62dFz6P864ZLlO+dyCg5EKEIFw77+wEGZE/rub0u/olWqJVbeC8OyFDbp8e0j/Lir5SciFCkBIXzoxMEzvuyop4jKS6LkRvruVb1FPruB37giIiI49/VoZ4CJGJd9RQNn9MyYUIQUoLIx/N12BTCZxu4HhGaOSx3YSt57yLXDUxj7L5PGsS6p1OPyq5EGGF8qAbwBjSz0fDT+GxDkQi4YzSngg/7E48Wsnjn1Us/bMSQW34NyUXIgQpcdkdhNPvCusUP/zEY7TJeiUoqlm2im9LrkwjnQxx/b8ruRAhSHem2LX61dF/BTIxXcIYeZDL70prS14RBEECHg45rZ9feDBL/ywOLMNlKLkQIUhvF+1YQ6QPRPzgEPhKfcjoK6VFJwQnW7WBccpX5jfUArvmc5RciBCkV1ee2SDNWpg40cl+V1qbxbmZ6iKxixLxfKXwIKDkQgRV5KRwg/sCMjD17eFcvyutXfTZN8303X6eRcmAD/5ZfK7+WtvgxfPAAWV+lghB4H/SXYTLcbpwunuvEDv+kCEqVcKZeAdRJDOd332pglpzveCAqi8WSuCKRKhZG50nYmyIDExcwmmgRnRaLnKN1DbwEBT/ehdI/ywj3RB1TSwHov5zBvVx9mwItpwiwgeHfLMHUUd4hjhG9akTiXiWr+FbvZf+WUZ74vh1sRyIIgiCHjhV4KPPmUc8XiIcu9vPeRdDPOJ+Pd0z2WI31lrqi+VAFEEGOvgOgnUP5QqnlsbGO6dKZIKy2u8qXt2xx5E08lORmIdDOvyz+CLac+jktjJiORBFkAG9fSJkpzObxOmY9/qH9bb4XWlFknPK98qJuWx787Xq75XzYPvggFKi6IDtnKee5LznkDNEJA9kXOIQ+F0tYYy+zqETFg2RoIdDCvMy6fLxcsrXCJawfXBAKVEEwQ8jBzuPIJx3j4+3iPwdh3iMUQmkmc9IzPPKe5z+WXw54c20y23Xqtl+Z4IM4vmn5gKnRGRYuljs++ASzodiLp1QDhL0KOMxmSl7cF+/I//ozLTJjmtHqth+J4KMUDnJDuXi1QG/q2wmv6v9R4KsMari6Wzk+M+VIYIrPJfIKV/9SKrZfieCdO+S5ZhQ+1j2nynWPrhkqRjGIAOUU4XTq3jqmBwqK+7UvU5tesr1QnoJ2H6sqCI4nnG1OrZCPd8nnom8GNGzCnquV57b1IJ4V3zjfGXZXJ+RqAcJezgE7hIIDyRFGwLxbF6VIHAIdIJw+10p8z44oX1qOnBO+V43BZ4HarXIY7EIxLN5VfjOHhoQDluxRdj7vU/3LEJmJS7hNDwundTKQcIeJO7hkB5iyDBlVA5HUZ4uA7Y+Vti8mqgSBGFPzhqsfoFaIVYcg98VVoY55GuR7WnrLp6hC4c+icpAwh4EkOASzsAWXDo5rRzYep6weTVRPyrOvOCs1P3zIFIHp98VZyQRNRC5j8FPDAuaHHL20GzHe2lztNNMGZMS2HpcgiDXH9MfuG7dLxV+V4jYwSGVwu/qX2Lfh5sEASTWfcUXg2jWVPmwHq//YePYkRlP4lohprxSNczinMN/TcxcNfHZWjwc2Y9z3vWumJArhhA8w1X2hqa4QNi42vRuWK24BMEJ08fGZ1a4AO734af46bT+PO4uGKZg7cONgqQ7B47ypEyAfxZIIqUzAslsPCFBcOuxe5qQcziwZrPwuzrsLL+rzl2kfgQJfDjJzenPpq6x+47CthMNr9CihARBvnEs1tkliMyBDEpc4pap3XjtRSIfLHByCPY5jB6S2plJjnZwlgHbho0nkoQEwYXXTrLvAe8aEak91t04kfKJfvtR+F199CXPRqRE9Vj5GwJKrFzPN8Egp3yje0uLbSclyMQR2dStNOlp0TUb+IbbHW+8K2f7XWmF6KW3+Z6hpozOoa4l1vel1ral8jzYNGw7mSRFK0sESbhKZJC1WqBsr648y/dNzW2E2SsvyBaxwImFTg7xZUn/rDCOWGeDbSeTpARBAXB6s3orLufU7lti3aOylmfsngxAO37nnPKdYUNf2oGJmTpgy8BBi2giCGIuTbNwyrevyJSEjElc4vaH81gcsNCJBU8OwZz/hWKolc4CW1ZGT0yEhSaCoIBbLi1IVI6p3xDCn8vvauuuZvrqe54hialGMV6MhU7OIeOsi7T9ezI2wVFF6bFlzQTB4t2Y0/j+5cOI5YgirxKROLjEa3ePMC4vi0AToRDPsHHMkOxO0TvC9Xj9HTasZyFaM0EA3K2X8cdcukxE4CjK16VG3D6sqAkRnj+8KD8eaSUsfHIJ514bLp3sKOe2y/XZsC7LPO/MHJH2jMcNJAwGZ4Yk7Bhs9tboKgxT+zvn3RGuJ/lp5p8F250wXN/zly6CoJfmziiM6jQzX5AZaQhTHC4MPzhdM8y0y6prO1xnePyzsOfn14xDW6vazFmuEdvNEA59uge2199fThu+Ne8iiyh2sZHsjAKCKOlrt5jXyWj9dl2H5KV9uvOsFx2paKVNO5LfcpGPnnt/0LqtTWLnpG7TMwzzqFMD9PKDZbqvN0SQz7c10awHjumuTF4gEUgVAkse6CJ80ZKvnMfqp3uIhQJQ0eRR+iuLrVx+lwjYgcCU0dmGyAHdDBEEF947u4hw65UiEXAyArDR/7yxyLCKhgmCcfBvLtY3ZWZYS3mhRMAgArBRM89shgkCfX9/ZYHwpzdVhMFmy8skAskRgG3CRs2IKesuyM2k+b8xfvsyo7i8ViKQDAHYJmzUjJi7WtQ8XURel8kjzXSBvNYKBGCTsE2zYpogUOCBW4rTblXWLPDyeusQgIcAbJJDWAhyYlkW/fssvhV2jobJMtIXAdgibJJDWAgCRRA1Y9wwfm9fjkbKMtIHAdggZwQXNoJgP8djt5eITLTJtzGmT3fJltqJAGwPNsi1twi6sxEEhXUVIVQe/X0JPkqRCNiOAGwPNsgprASBYggEjMjsUiQCdiIAm0sUhNqoLuwEgSJ/mFNEp5/Mu2/EaAPldd5HALYGm7NCLCEIgr89dXcpIVKiFImAlQjAxmBrXAEHY3W1zIJPFJFQFs8tsT22b2wD5XfvIoBgg7Ax2JpVYhlBoPBYkbzlP26Q6yNWdV66lwvbgo1ZKZYSBIrfdEkB67y0lWDIst2DANY6YFtWi+UEQQPm31SkKQ6q1Y2V5XsDAYSphU3ZIbYQBDFQMVbEfmopEgEzCMCGnhC2pCWurpl6wtfaQhBUhigaz/2hC/Xrad0DVbhR8t2bCMB2YEPxMtJa0WrbCALly4qz6IUFZdS7mySJFZ3p5TJhM7Ad2JCdYitB0DAkt39hQReRONH2qu3EVdbFiABsBTYD27FbUmKlv+rma/83sCMxj92Ayvp4EYCN4M4Bm0mFGIqLxaXovsNBmv3QMZHN1Z2JNrlwkOWoI9Cra8dow0zQBfWStR9NKUGg5k/lrTRnYTntOShJor3bvH8mHsifn1/GtvHJKGIpJwgUL69qpZsePkbbfuCJO2sUDHmdMxAYcpKPnruvi+0P5GqtdwRBoFhdY4juXFTp+sy0aiDLY9oROE+kZn5iXkn7soD2q6w70zEEQRNbRYT2B5+t9nyUduu6090lI8vx/TcX2bYIqAUtRxEkrPAzb9TSY0tqSH/c+XAJ8t1NCIjd2nSPCLRwy2XW+1bpxcWRBEEj1orw+PMWV3gqW63ezkmH80sKMmjR3FKRxNVar1yjWDqWIGjQgaNBuv3xSvqGKU+4UZDkddYggFyBT91dQr26pmaNQ0urHE0QNKCpuY0e+lsVvbram7kHtXSSF8+5dlIuLfhtMWUHnB0Fx/EECRvHO5810H1/qaKqWvuyEoXrlu98CBSLIdXDvyumqWPMhwXl0yp+Sa4hCJpw6Fgr3fNkJa3/2vup1uJ3mXt/GTs0QI/dUZISnyqjqLmKIGgkUio+v7Ke/ri0hhqa5N3EaMfbeV1udgbdNbOQ5kzLYw3qZkcbXEeQMCh4gJ//12r6mDF3eLhs+c6HwPjh2bTw1iJHP4gnaq1rCRJu1PI19fToCzV0rDoUPiTfHYAAwvHcO7uQrpjg7iCCricIbKG6LkRPvlZLL75dR0Hp85hSevjElo0bL8qnO64uoKL8lOymYG2/JwgSRmT3wSA98ny19OcKA2LzO/yoEOGwf0/nrmvohcRTBAk3HknqF71SQ1t2tYQPyXcLERg2wE/zriukcx26Gm6m6Z4kSBiQDzY10uJlNbRtj3SjD2PC+Y4II3NnFNL5I3I4i3VUWZ4mCJDGtPCqDU30tHCA3LxT3lE4rG/4QD/dJhwLJ4/Kdt20rd72e54gSkA27WimZ9+so/c2NEpPYSUwGj7D43bKqBy6+dJ8GjEofTKJpRVBwnaw91CQXn63npavaZDTw2FQ4rxjuvaKCbntOV/69vDOw3ec5nY6nJYECaPQHBTDr88badnqevrkK+m+EsYF7+ecHqAZk/Jo8ugcCvic7VCo1Jv7c1oTRAnmwZ9bacUnDbRifSN99X16PqsgEc30sTnt+cV7WphSQIm70z9Lgqj0EMIRrRREWfVFI20VU8Uhjy7SI7/GGWKKdrJImzf9nJyUxZ5S6QLHHJIESdIVVbWh9t2NH33ZRGu3NNHRSnezpWtJpkjXnU1Y1MMuvuIC9692J+lCUz9LguiEDw/4G7c300YxI7ZpRwvt2u/sNZYBvX1i1slPI8XM08jBAUrHB22dXRx1uiRIFBz6v9TUh2jnviBt39fS8b63hXYfCNq+lx57u/v38tHgvn4a2Ee89+l4L8yTdwj9vXr8CkmQ41iwfqptCLWHVN1/pJXwOlLRShU1ocirsqaN6sQ5zeIG1CJm0zCj1vLLzcgvZlMxc+QXr4D4nJ+bSSWFGVRamBl5dSvNao+Sj6jnCNFZIM6Rwo/A/wPEgyy/lRCYSgAAAABJRU5ErkJggg==`,
}
```

### CODE REVIEW

File: src/webchat-ui/components/WebchatUI.tsx
Comment (fabclj) [inline src/webchat-ui/components/WebchatUI.tsx:1]: Changing the indentation is losing the direct changes. Review this file is not feasible like that.

File: src/webchat-ui/components/WebchatUI.tsx [hunk 1]
```diff
@@ -1,871 +1,938 @@
...
+					onEmitAnalytics={onEmitAnalytics}
+				/>
+			);
+
+		const handleOnClose = () => {
+			onSetShowPrevConversations(false);
+			onSetShowHomeScreen(true);
+		};
+
+		const handleOnGoBack = () => {
```
Comment (fabclj) [inline src/webchat-ui/components/WebchatUI.tsx:762]: This way the back arrow is always going to HomeScreen. This is fine for the close icon, but this should be back to the previous view. For example: if I reach the chat from PrevConversationList, as user I expect to to be back on list.

Comment (kwinto) [reply src/webchat-ui/components/WebchatUI.tsx:762]: Actually close icon should always close the widget, not go back to home. I will check.

**Comment by fabclj** [comment]
Need to handle the :focus on buttons. It appears a strange circle.
Screenshot from 2023-12-13 10-01-41

File: src/webchat-ui/components/WebchatUI.tsx (hunk 1 above)
Comment (fabclj) [reply src/webchat-ui/components/WebchatUI.tsx:762]: I mean the arrow back. The close button behavior is fine to me. But the arrow back should go back to the previous screen, not just to Home Screen.

**Comment by fabclj** [comment]
I would suggest to add a :hover effect on buttons. We already did something similar on Lightbox component for downloadable image. We should approach on same way.
Screenshot from 2023-12-13 10-08-43

File: src/webchat-ui/components/WebchatUI.tsx
Comment (kwinto) [reply src/webchat-ui/components/WebchatUI.tsx:1]: Sry, it slipped through by resolving merge conflict

File: src/webchat-ui/components/WebchatUI.tsx (hunk 1 above)
Comment (kwinto) [reply src/webchat-ui/components/WebchatUI.tsx:762]: This is out of scope. We will implement a navigation solution later, as per your suggestion earlier.

File: src/webchat-ui/components/presentational/Header.tsx [hunk 2]
```diff
@@ -1,82 +1,157 @@
...
-    boxShadow: '0 5px 18px 0 rgba(0, 0, 0, 0.08), 0 5px 32px 0 rgba(0, 0, 0, 0.08), 0 8px 58px 0 rgba(0, 0, 0, 0.08)',
-    zIndex: 2,
-    minHeight: 'auto',
-    height: theme.unitSize * 7,
-    flexBasis: theme.unitSize * 7,
-    flexShrink: 0,
-    fontSize: 16,
-    fontWeight: 700
-}))
+const HeaderBar = styled.div(({ theme }) => ({
```
Comment (fabclj) [inline src/webchat-ui/components/presentational/Header.tsx:9]: Last but not least, I would suggest to implement the new Typography component here.

Comment (kwinto) [reply src/webchat-ui/components/presentational/Header.tsx:9]: Typography is coming in another open PR from Sushmita, we can add a comment there after this is merged

Comment (fabclj) [reply src/webchat-ui/components/presentational/Header.tsx:9]: It was already merged. We could anyway do a second round.

**Comment by kwinto** [comment]
> I would suggest to add a :hover effect on buttons. We already did something similar on Lightbox component for downloadable image. We should approach on same way. Screenshot from 2023-12-13 10-08-43

Figma designs do not have hover state at all (only focus state). I would be very conscious regarding adding new styles which are not in the reference design. 

I will raise the question to the PO.

**Review by fabclj — APPROVED** [review]

**Pull request merged**